lib/diff.js

```diff
@@ -127,7 +142,15 @@ Diffing.prototype.diffRules = function(oBaseRules, oCompareRules) {
 
 	// Add all leftover compare nodes to stack
 	for (; iCompareNode < oCompareRules.length; iCompareNode++) {
-		this.oStack.stylesheet.rules.push(oCompareRules[iCompareNode]);
+		if (oCompareRules[iCompareNode]) {
+			this.oStack.stylesheet.rules.push(oCompareRules[iCompareNode]);
+		}
+	}
+
+	for (; iBaseNode < oBaseRules.length; iBaseNode++) {
+		if (oBaseRules[iBaseNode]) {
+			this.oStack.stylesheet.rules.push(oBaseRules[iBaseNode]);
```

Comment:
Afaik only the compareRules should be added

Reply:
ok